code/game/objects/random/circuitboards.dm

```diff
@@ -1,63 +1,15 @@
-/obj/random/circuitboard
+/obj/item/weapon/electronics
+	spawn_tags = SPAWN_TAG_ELECTRONICS
+	rarity_value = 20
+	spawn_frequency = 10
+	bad_types = /obj/item/weapon/electronics
```

Comment:
We would like if you can find the original declaration to put these variables in /obj/item/weapon/electronics at proper file for circuit boards.